deps.ffmpeg/70-nv-codec.zsh

```diff
@@ -11,11 +11,21 @@ local targets=('windows-x*')
 
 ## Build Steps
 setup() {
+  if [[ ${target} == "windows-arm64" ]]; then 
```

Comment:
This should not be necessary as the `targets` variable above explicitly limits the dependency to `windows-x*` targets, which should not match `windows-arm64`.